tests/clv/test_utils.py

```diff
@@ -136,79 +122,31 @@ def transaction_data() -> pd.DataFrame:
 
 
 class TestCustomerLifetimeValue:
+    @pytest.mark.parametrize(
+        "t, discount_rate, expected",
+        [
+            (1, 0, np.array([15.91086, 0.7538, 0])),
+            (1, 1, np.array([7.95543, 0.3769, 0])),
+            (2, 0, np.array([26.21316, 1.28481, 0])),
+            (2, 1, np.array([10.531, 0.50965, 0])),
+        ],
+    )
     def test_customer_lifetime_value_with_known_values(
-        self, test_summary_data, fitted_bg
+        self, test_summary_data, fitted_bg, t, discount_rate, expected
     ):
-        # Test borrowed from
-        # https://github.com/CamDavidsonPilon/lifetimes/blob/aae339c5437ec31717309ba0ec394427e19753c4/tests/test_utils.py#L527
-
-        t = test_summary_data.head()
+        data = test_summary_data.head(3)
 
-        expected = np.array([0.016053, 0.021171, 0.030461, 0.031686, 0.001607])
```

Comment:
I insist on keeping the original test with the original values. This is a good independent source that was being used in a third-library.

We can hardcode the old `test_summary_data.head()` if the point was to remove the utility dataset

Reply:
I hardcoded the old test data, but am keeping the pytest parametrization to reduce code repetition.